datadog/fwprovider/resource_datadog_team.go

```diff
@@ -75,6 +76,10 @@ func (r *teamResource) Schema(_ context.Context, _ resource.SchemaRequest, respo
 				Required:    true,
 				Description: "The name of the team.",
 			},
+			"avatar": schema.StringAttribute{
+				Required:    false,
```

Comment:
Hello @jmcarp. Sorry for the late reply.
This should be:
```suggestion
				Optional:    true,
```

Also please regenerate the test cassettes (see [DEVELOPMENT.md](https://github.com/DataDog/terraform-provider-datadog/blob/master/DEVELOPMENT.md#testing-the-provider)) so we can approve this PR.